h/db/__init__.py

```diff
@@ -153,6 +135,27 @@ def _maybe_create_default_organization(engine, authority):
     session.commit()
     session.close()
 
+    return default_org
```

Comment:
`_maybe_create_default_organization()` returns the default org whether it already existed or was just created